src/acl/access-groups.ts

```diff
@@ -0,0 +1,274 @@
+      const combo = kToCombo(comboIndex)
+      if ((this.controller.isEditable && RECOMMENDED[comboIndex]) || this.byCombo[combo]) {
+        groups.push(this.renderGroup(comboIndex, combo))
+      }
+    }
+    return groups
+  }
+
+  private renderGroup (comboIndex: number, combo: string): HTMLElement {
+    const groupRow = this.controller.dom.createElement('div')
+    groupRow.classList.add(this.controller.classes.accessGroupListItem)
+    widgets.makeDropTarget(groupRow, (uris) => this.handleDroppedUris(uris, combo)
+      .then(() => this.controller.render())
+      .catch(error => this.controller.renderStatus(error)))
+    const groupColumns = this.renderGroupElements(comboIndex, combo)
+    groupColumns.forEach(column => groupRow.appendChild(column))
+    return groupRow
+  }
+
+  private renderGroupElements (comboIndex, combo): HTMLElement[] {
+    const groupName = this.controller.dom.createElement('div')
+    groupName.classList.add(this.controller.classes.group)
+    groupName.classList.toggle(this.controller.classes[`group-${comboIndex}`], this.controller.isEditable)
+    groupName.innerText = COLLOQUIAL[comboIndex] || ktToList(comboIndex)
+
+    const groupAgents = this.controller.dom.createElement('div')
+    groupAgents.classList.add(this.controller.classes.group)
+    groupAgents.classList.toggle(this.controller.classes[`group-${comboIndex}`], this.controller.isEditable)
+    const groupAgentsTable = groupAgents.appendChild(this.controller.dom.createElement('table'))
+    const combos = this.byCombo[combo] || []
+    combos
+      .map(([pred, obj]) => this.renderAgent(groupAgentsTable, combo, pred, obj))
+      .forEach(agentElement => groupAgentsTable.appendChild(agentElement))
+
+    const groupDescription = this.controller.dom.createElement('div')
+    groupDescription.classList.add(this.controller.classes.group)
+    groupDescription.classList.toggle(this.controller.classes[`group-${comboIndex}`], this.controller.isEditable)
+    groupDescription.innerText = EXPLANATION[comboIndex] || 'Unusual combination'
+
+    return [groupName, groupAgents, groupDescription]
+  }
+
+  private renderAgent (groupAgentsTable, combo, pred, obj): HTMLElement {
+    const personRow = widgets.personTR(this.controller.dom, ACL(pred), sym(obj), this.controller.isEditable ? {
+      deleteFunction: () => this.deleteAgent(combo, pred, obj)
+        .then(() => groupAgentsTable.removeChild(personRow))
+        .catch(error => this.controller.renderStatus(error))
+    } : {})
+    return personRow
+  }
+
+  private async deleteAgent (combo, pred, obj): Promise<void> {
+    const combos = this.byCombo[combo] || []
+    const comboToRemove = combos.find(([comboPred, comboObj]) => comboPred === pred && comboObj === obj)
+    if (comboToRemove) {
+      combos.splice(combos.indexOf(comboToRemove), 1)
+    }
+    await this.controller.save()
+  }
+
+  public updateStore (store: IndexedFormula): void {
```

Comment:
Name of function - 'loadAndConvertACL' ?    It does load the ACL into the store, but also mainly into the internal byCombo form in this object.   Prefer to use 'update' for changes going back to the pod, 'load' for things coming in the read direction.

Reply:
I realized this might be better to fix with a setter. I've committed a solution using this approach in the latest commit.